integrationtests/testground/go.mod

```diff
@@ -10,13 +10,15 @@ require (
 	github.com/ipfs/go-ipld-cbor v0.0.5
 	github.com/ipfs/go-log/v2 v2.1.3
 	github.com/libp2p/go-libp2p-core v0.8.5
-	github.com/multiformats/go-multiaddr v0.3.1
-	github.com/multiformats/go-multihash v0.0.14
-	github.com/prometheus/client_golang v1.9.0
+	github.com/multiformats/go-multiaddr v0.3.3
+	github.com/multiformats/go-multihash v0.0.15
+	github.com/prometheus/client_golang v1.10.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/node_exporter v1.1.2
 	github.com/testground/sdk-go v0.2.7
 	github.com/textileio/go-threads v1.1.0-rc1.0.20210609142634-18b4ea73088f
 	google.golang.org/grpc v1.37.0
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 )
+
+replace github.com/textileio/go-threads v1.1.0-rc1.0.20210609142634-18b4ea73088f => github.com/mcrakhman/go-threads v1.1.0-rc1.0.20210626214637-1a88e4ef1d3b
```

Comment:
This should be change definitely, we can also get rid of replace, but still to do this we need to have a merged commit in upstream `go-threads` repo. Not sure how to do this right now. Maybe have a follow up PR after we merge this one?